src/fondant/__init__.py

```diff
@@ -1,6 +1,12 @@
 import logging
 
+import dask
+
 logging.basicConfig(
     format="[%(asctime)s | %(name)s | %(levelname)s] %(message)s",
     level=logging.INFO,
 )
+
+
+dask.config.set({"dataframe.convert-string": False})
```

Comment:
I am not 100% if this guarantees that it's applied to the rest of the modules. I may have been wrong before. The best place for it indeed might be in the executor since that's the starting point and it will apply globally to all other modules called within it. 

More details here: 
![image](https://github.com/ml6team/fondant/assets/47530815/3d51b4a3-29cc-497c-bb21-6c0b7aadb9a6)

Reply:
Yes indeed. See my comment above. Will move both configuration to the executer.